internal/ingress/metric/collectors/socket.go

```diff
@@ -30,11 +30,10 @@ import (
 )
 
 type upstream struct {
-	Endpoint       string  `json:"endpoint"`
 	Latency        float64 `json:"upstreamLatency"`
 	ResponseLength float64 `json:"upstreamResponseLength"`
 	ResponseTime   float64 `json:"upstreamResponseTime"`
-	Status         string  `json:"upstreamStatus"`
+	//Status         string  `json:"upstreamStatus"`
```

Comment:
This will be brought back as part of the solution for https://github.com/kubernetes/ingress-nginx/issues/3450